packages/grid/src/features/standard-table/components/StandardTableRow.tsx

```diff
@@ -73,8 +77,11 @@ export const StandardTableRow = React.memo(function StandardTableRow<TItem>({
       <TableRow
         height={"var(--current-row-height)"}
         width={"100%"}
-        borderTop={isExpanded ? tableBorderExpanded : tableBorder}
-        background={background ?? "white"}
+        borderTop={tableBorder}
+        borderLeft={isExpanded ? tableBorderExpanded : undefined}
```

Comment:
Maybe use outline or add some margin when not expanded.
Expanding shifts the row 2px to the right

Reply:
Or transparent border